lib/src/models/view/sellerprofile.dart

```diff
@@ -0,0 +1,54 @@
+import "package:btc_market/data.dart";
+import "package:btc_market/models.dart";
+import "package:btc_market/src/data/category.dart";
+import "dart:collection";
+
+class SellerProfileViewModel extends ViewModel {
+  // The profile variable will hold the object of Seller Profile
+  /// The profile will get the seller profile info from model of seller profile.
+  SellerProfile? get profile => models.user.sellerProfile;
```

Comment:
I tried doing this, but since the type of models.user.sellerProfile is SellerProfile?, it cannot be assigned to SellerProfile. The models.iuser.sellerProfile is optional, so it's not allowing to assign me the value to SellerProfile if I keep it non null.

Reply:
Thank you! This is done.